pkg/antctl/handlers/version.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2019 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package handlers
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/vmware-tanzu/antrea/pkg/monitor"
+)
+
+var _ Factory = new(Version)
+
+// VersionResponse describes the response of the version command.
+type VersionResponse struct {
+	AgentVersion      string `json:"agentVersion,omitempty" yaml:"agentVersion,omitempty"`
```

Comment:
If this is returned by the server, it should include a single version field? Not sure if I misunderstood.

Reply:
Replied in prior comments, thanks.